third_party/ant/LICENSE

```diff
@@ -0,0 +1,272 @@
+/*
```

Comment:
per malte's old instructions, something like this should be done in its own commit or its own Pull Request

Reply:
This is in a separate commit: https://github.com/ampproject/amphtml/pull/23434/commits/b354627f05636b1b8f81949d9f16ab29df2420c4

Did you mean it should be checked in first and then used in a follow up PR? (We haven't done that with most other third party libraries.)

Reply:
if its in a separate commit, just make sure to not squash when you merge then

Reply:
That'll create too many commits on `master`. Let me fork it off into its own PR.

Reply:
Done.